### PR TITLE
Add license and maintainers file

### LINKED ISSUE #1: LICENSE file
hi guys...

Looks good!

Could you do me a kindness and add a `LICENSE` file.  It will save me a trip to the lawyers.

Thanks!

nickg

### PR DESCRIPTION
**Pull request by thaJeztah**
fixes https://github.com/docker/go-healthcheck/issues/1

ping @diogomonica @stevvooe PTAL

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# Contributing to Docker open source projects
+
+Want to hack on this project? Awesome! Here are instructions to get you started.
+
+This project is a part of the [Docker](https://www.docker.com) project, and follows
```
Comment (stevvooe) [inline CONTRIBUTING.md:5]: Did we ever merge the stuff I added to this?

Comment (thaJeztah) [reply CONTRIBUTING.md:5]: Which repo was that in? Can't find a PR in the docker/opensource repo

Comment (stevvooe) [reply CONTRIBUTING.md:5]: Looks like containerd. https://github.com/docker/containerd/blob/master/CONTRIBUTING.md#successful-changes

Let me know and I'll submit a PR.

**Comment by stevvooe** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (thaJeztah) [reply CONTRIBUTING.md:5]: Never hurts; looks like good things to mention

**Pull request merged**